sdks/python/apache_beam/runners/interactive/interactive_environment.py

```diff
@@ -43,6 +43,55 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+# By `format(customized_script=xxx)`, the given `customized_script` is
+# guaranteed to be executed within access to a jquery with datatable plugin
+# configured which is useful so that any `customized_script` is resilient to
+# browser refresh. Inside `customized_script`, use `$` as jQuery.
+_JQUERY_WITH_DATATABLE_TEMPLATE = """
+        if (typeof window.jquery341 == 'undefined') {{
```

Comment:
what is jquery341 ?

Reply:
jquery341 is probably not a good name. What would happen if we upgrade to a different jquery version? Maybe it would be better to call it jquery_singleton or something like that?

Reply:
I've changed it to `interactive_beam_jquery`.